lesson02/script.js

```diff
@@ -9,4 +9,4 @@ img.src = 'bookmark.png';
 img.alt = 'ブックマーク';
 
 const ul = document.getElementById('js-list');
-ul.appendChild(li).appendChild(a).prepend(img);
+ul.appendChild(li).appendChild(a).insertAdjacentElement( 'afterbegin', img ) ;
```

Comment:
Change the method of adding img from prepend to insertAdjacentElement,
because insertAdjacentElement is more common than prepend.